armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -67,11 +67,11 @@ def __init__(self, r, cs):
     @staticmethod
     def specifyInputs(cs):
         files = {
-            cs.settings[label]: [
-                cs[label],
+            cs.settings[setting].label: [
+                cs[setting],
             ]
-            for label in ["shuffleLogic", "explicitRepeatShuffles"]
-            if cs[label]
+            for setting in ["shuffleLogic", "explicitRepeatShuffles"]
+            if cs[setting]
```

Comment:
Hecka minor, but while "setting" is way better than "label", it still seems like it might refer to a `Setting` instance. "settingName"?